lib/widgets/layouts/agruments/app_adaptive_scaffold_body_args.dart

```diff
@@ -3,12 +3,15 @@ import 'package:matrix/matrix.dart';
 
 abstract class AbsAppAdaptiveScaffoldBodyArgs extends Equatable {
   final Client? newActiveClient;
+  final String? roomIdFromNoti;
```

Comment:
Why we need put it here?

Reply:
you're right, no need to put it in abstract class